### ISSUE TITLE
Symfony 4 : User Notice: Required @OA\PathItem() not found

### ISSUE THREAD

**Issue by mohamed-jebri** [nelmio/NelmioApiDocBundle#1656]
Hi,
I m using symfony 4.4 / nelmio api doc bundle 4.0 / swagger-php 3.0.4
When i try with a first example i get this error : User Notice: Required @OA\PathItem() not found

I tried with examples :
https://github.com/zircote/swagger-php/blob/3.0.0/Examples/example-object/example-object.php
https://github.com/zircote/swagger-php/blob/3.0.0/Examples/using-refs/ProductController.php

and when using ./vendor/bin/openapi src/ i got this output : 

openapi: 3.0.0
paths:
'/products/{product_id}':
get:
tags:
- Products
operationId: 'App\Controller\ProductController::products'
responses:
default:
description: 'successful operation'
content:
application/json:
schema:
$ref: '#/components/responses/product'
parameters:
-
$ref: '#/components/parameters/product_id_in_path_required'
components:
securitySchemes:
api_key:
type: apiKey
in: header

my example : 

`namespace App\Controller;

use OpenApi\Annotations as OA;

class ProductController
{

    /**
     * @OA\Get(
     *   tags={"Products"},
     *   path="/products/{product_id}",
     *   @OA\Response(
     *       response="default",
     *       description="successful operation",
     *       @OA\JsonContent(ref="#/components/responses/product")
     *   )
     * )
     */
    public function getProduct($id)
    {
    }

    }`

**Comment by strifel** [comment]
Similar Issue here.
As soon as I put 
```
nelmio_api_doc:
    areas:
        path_patterns: # an array of regexps
            - ^/api(?!/doc$)
        host_patterns:
            - ^api\.
```
or similar in the config file I get:
`Undefined error occured. User Notice: Required @OA\\PathItem() not found on line 39 in /var/www/symfony/vendor/zircote/swagger-php/src/Logger.php'`

**Comment by cybervoid** [comment]
I have Symfony 5, got the same error and it went away by clearing symfony's cache and running composer dump-autoload

**Comment by strifel** [comment]
I just found out I do not have this issue when running in prod environment

**Comment by GuilhemN** [comment]
Fixed in #1671, thanks for the report!
